backend/docs/API_Specifications/facility-v2.yaml

```
@@ -0,0 +1,382 @@
+          type: string
+          description: Region code where the facility is located. Eg. rural or urban
+          example: RURAL
+          enum: 
+            - RURAL
+            - URBAN
+        address:
+          $ref: 'https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/Address'
+        facility_details:
+          type: jsonb
+          description: A store of key value pairs per facility type. The exact set of key value pairs will be specific to the facility type. The schema can be defined as a master schema. Code should validate the payload against the appropriate schema as per the facility type.
+        wfStatus: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)